core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1933,29 +1946,43 @@ private[spark] class DAGScheduler(
       maybeEpoch = None)
   }
 
+  /**
+   * Handles removing an executor from the BlockManagerMaster as well as unregistering shuffle
+   * outputs for the executor or optionally its host.
+   *
+   * The fileLost parameter being true indicates that we assume we've lost all shuffle blocks
```

Comment:
Nit: what about documenting the params (at least the `fileLost` and `maybeEpoch`) with a "@param" scaladoc annotation?

Reply:
Added parameters to doc comment.